src/components/UiCardList/constants.ts

```
@@ -1,67 +1,21 @@
+
 import Diamond from '../../features/landing/assets/svg/possibilities/diamond.svg';
 import Ruby from '../../features/landing/assets/svg/possibilities/ruby.svg';
 import SmallDiamond from '../../features/landing/assets/svg/possibilities/smallDiamond.svg';
 import SmallRuby from '../../features/landing/assets/svg/possibilities/smallRuby.svg';
```

ESLint

[error] There should be no empty line within import group

(import/order)